apps/api/src/app/support/support.controller.ts

```diff
@@ -37,4 +38,29 @@ export class SupportController {
       })
     );
   }
+
+  @UserAuthentication()
+  @Post('mobile-setup')
+  async mobileSetup(@UserSession() user: UserSessionData) {
+    if (!process.env.NOVU_INTERNAL_SECRET_KEY) {
+      throw new Error('NOVU_INTERNAL_SECRET_KEY is not set');
```

Comment:
please add this validation in the apps/api/src/config/env.validators.ts as well. 
we should to assume that this env is not undefined on this stage.